### PR TITLE
Static page deployment, dockerization and cloud configuration

### PR DESCRIPTION
**Pull request by RyanKelvinFord**
This is a dockerized version of a static web page that has been configured and run locally.

### CODE REVIEW

File: deployment/plat-app-services/static-page/base.yaml [hunk 1]
```diff
@@ -0,0 +1,40 @@
...
+    matchLabels:
+      app: static-page
+  template:
+    metadata:
+      labels:
+        app: static-page
+    spec:
+      containers:
+        - image: "ryankford/kitt4sme-static-page:1.0"
+          imagePullPolicy: Always
```
Comment (c0c0n3) [inline deployment/plat-app-services/static-page/base.yaml:36]: shouldn't this be `IfNotPresent`?

File: deployment/plat-app-services/static-page/manifest.yaml [hunk 2]
```diff
@@ -0,0 +1,40 @@
...
+spec:
+  ports:
+  - name: http
+    nodePort: 30000
+    port: 80
+    protocol: TCP
+    targetPort: 80
+  selector:
+    app: static-page
+  type: NodePort
```
Comment (c0c0n3) [inline deployment/plat-app-services/static-page/manifest.yaml:16]: We should avoid node ports if possible. But it just occurred to me this file isn;t actually included in the Kustomize build, so I guess you were just testing?

**Review by c0c0n3 — COMMENTED** [review]
@RyanKelvinFord good stuff! if there's no particular reason to pull images every time the yaml changes, then should we use cached images? see my comment. thanks alot!!

File: deployment/plat-app-services/static-page/base.yaml (hunk 1 above)
Comment (RyanKelvinFord) [reply deployment/plat-app-services/static-page/base.yaml:36]: I will change that.

File: deployment/plat-app-services/static-page/manifest.yaml (hunk 2 above)
Comment (RyanKelvinFord) [reply deployment/plat-app-services/static-page/manifest.yaml:16]: Yes apologies that should have been removed, will just confirm.

Comment (RyanKelvinFord) [reply deployment/plat-app-services/static-page/manifest.yaml]: The file manifest.yaml should not have been included it was from an earlier attempt at this. It has been removed now.

File: deployment/plat-app-services/static-page/base.yaml (hunk 1 above)
Comment (RyanKelvinFord) [reply deployment/plat-app-services/static-page/base.yaml]: This change has been made.

**Comment by RyanKelvinFord** [comment]
I have merged it as it seemed it was accepted, will now wait for it to be deployed and then test it.

**Comment by RyanKelvinFord** [comment]
The application is not being picked up automatically, I give it some time and then start it manually.

**Pull request merged**